source/GameData.cpp

```diff
@@ -456,6 +459,7 @@ void GameData::Preload(const Sprite *sprite)
 void GameData::FinishLoading()
 {
 	spriteQueue.Finish();
+	maskManager.ScaleMasks();
 }
```

Comment:
Preload/FinishLoading also seems to be strictly used for landscape images, so really it's only a coincidence that scaling the masks here is done when it's needed for scaled collision masks to work, and there's probably some more appropriate place elsewhere that this could be done.

Reply:
I've moved the scaling of masks to after SpriteQueue is done loading at the start of the game (when GameData::Progress is finished). The only sprites that don't get loaded at the start of the game are the landscape images, which don't have masks, so we shouldn't need to scale any new masks after the game starts.